.nf-core.yml

```diff
@@ -15,9 +17,21 @@ lint:
     - .github/PULL_REQUEST_TEMPLATE.md
     - .github/workflows/branch.yml
     - assets/email_template.txt
+    - assets/email_template.html
     - assets/sendmail_template.txt
-    - lib/NfcoreTemplate.groovy
     - .prettierignore
+    - assets/email_template.txt
   nextflow_config:
     - manifest.name
     - manifest.homePage
+nf_core_version: 3.2.0
+repository_type: pipeline
+template:
+  author: Damon-Lee B Pointon (@DLBPointon)
+  description: A simple pipeline to generate pretext files for genomic curation.
+  force: false
+  is_nfcore: true
```

Comment:
You should specify `is_nfcore: false` here and re-sync the template - it pulls in a bunch of nf-core infrastructure stuff.

Reply:
Done